baseclasses/testing/pyRegTest.py

```diff
@@ -343,95 +344,6 @@ def _add_dict(self, dict_name, d, full_name, db=None, **kwargs):
             else:
                 self._add_values(key, d[key], rtol=rtol, atol=atol, db=db[dict_name], full_name=full_name)
 
-    # =============================================================================
```

Comment:
Is this just deprecated?

Reply:
This was a helper function to convert the old-style ref files to the new style. AFAIK we have converted everything a long time ago, so this is no longer needed.

Reply:
I agree. We should be good to delete this no that it is no longer needed.